### ISSUE TITLE
Increase contrast of focus rings around interactive elements

### ISSUE THREAD

**Issue by jenewland1999** [HaveIBeenPwned/ux-rebuild#27]
As the screenshot below shows, the focus rings around various interactive elements are quite difficult to see due to the similar colour to the background. From an accessibility perspective, it also doesn't meet the minimum WCAG 2.2 AA contrast ratio either. 

I can't recall all the colours in your new brand colour palette, but I imagine a much brighter shade of blue, or if you're feeling more adventurous a bright yellow might look quite good.

![Image](https://github.com/user-attachments/assets/6b7c0444-5416-4e11-ac55-db1a25bdd512)
